src/applications/find-va-forms/reducers/findVAFormsReducer.js

```diff
@@ -22,9 +21,10 @@ export default (state = initialState, action) => {
       return { ...state, fetching: false };
     }
     case FETCH_FORMS_SUCCESS: {
-      // Normalize the forms data we get back from the API resopnse.
-      const results = normalizeFormsForTable(action.response);
```

Comment:
Moving logic out of the reducer to keep it cleaner 👍 The normalization now happens higher up in [this file](https://github.com/department-of-veterans-affairs/vets-website/pull/11312/files?file-filters%5B%5D=.js&file-filters%5B%5D=.lock#diff-8b5f2b0101273662305bd5fda858ed30).

Reply:
I think reducers are actually really underused and that this is a perfectly fine use case for a [reducer](https://redux.js.org/basics/reducers) -> 

> Reducers specify how the application's state changes in response to actions sent to the store. Remember that actions only describe what happened, but don't describe how the application's state changes.

To me, this data normalization is a response to the action - the action is dispatched with the original data structure, and the reducer responds by transforming it into application state.

Reply:
I understand where you're coming from 👍In the past, I've experienced the opposite where reducers have a ton of logic and side-effect logic in each case statement, which is what I want to avoid as it can become extremely confusing and side-effect logic should occur in some type of Redux middleware if it has to. `normalizeFormsForTable` doesn't introduce side-effects so we should be all good either way :)

Reply:
Works for me, whichever you prefer!